packages/frontend/app/routes/reports/curriculum.js

```diff
@@ -0,0 +1,54 @@
+import Route from '@ember/routing/route';
+import { service } from '@ember/service';
+import { DateTime } from 'luxon';
+
+export default class ReportsCurriculumRoute extends Route {
+  @service session;
+  @service store;
+  @service graphql;
+  @service currentUser;
+
+  queryParams = {
+    courses: {
+      replace: true,
+    },
+  };
+
+  beforeModel(transition) {
+    this.session.requireAuthentication(transition, 'login');
+  }
+
+  async model() {
+    const schools = await this.store.findAll('school');
+    const threeYearsAgo = DateTime.now().year - 3;
+    // Limit query to surounding years
```

Comment:
_pedantic jerk alert_ - this is a typo.

```suggestion
    // Limit query to surrounding years
```

Reply:
✅, didn't commit yours because I was already rebased locally. Fixed.